[info] Running ex2.Ex1 
[debug] Waiting for threads to exit or System.exit to be called.
[debug] Waiting for thread run-main-2 to terminate.
[debug]   Classpath:
[debug] 	/home/yuwki0131/Dropbox/gwork/open-my-doc/src/tryscalaz/target/scala-2.12/classes
[debug] 	/home/yuwki0131/.ivy2/cache/org.scala-lang/scala-library/jars/scala-library-2.12.4.jar
[debug] 	/home/yuwki0131/.ivy2/cache/org.scalaz/scalaz-core_2.12/bundles/scalaz-core_2.12-7.3.0-M18.jar
[error] (run-main-2) java.lang.NoSuchMethodException: ex2.Ex1.main([Ljava.lang.String;)
java.lang.NoSuchMethodException: ex2.Ex1.main([Ljava.lang.String;)
	at java.lang.Class.getMethod(Class.java:1786)
	at sbt.Run.getMainMethod(Run.scala:73)
	at sbt.Run.run0(Run.scala:60)
	at sbt.Run.sbt$Run$$execute$1(Run.scala:51)
	at sbt.Run$$anonfun$run$1.apply$mcV$sp(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Run$$anonfun$run$1.apply(Run.scala:55)
	at sbt.Logger$$anon$4.apply(Logger.scala:84)
	at sbt.TrapExit$App.run(TrapExit.scala:248)
	at java.lang.Thread.run(Thread.java:748)
[debug] 	Thread run-main-2 exited.
[debug] Interrupting remaining threads (should be all daemons).
[debug] Sandboxed run complete..
java.lang.RuntimeException: Nonzero exit code: 1
	at scala.sys.package$.error(package.scala:27)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$44$$anonfun$apply$45$$anonfun$apply$46.apply(Defaults.scala:793)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$44$$anonfun$apply$45$$anonfun$apply$46.apply(Defaults.scala:793)
	at scala.Option.foreach(Option.scala:236)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$44$$anonfun$apply$45.apply(Defaults.scala:793)
	at sbt.Defaults$$anonfun$runMainTask$1$$anonfun$apply$44$$anonfun$apply$45.apply(Defaults.scala:791)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
[error] (compile:runMain) Nonzero exit code: 1
